diff --git a/plugins/gtkui/callbacks.c b/plugins/gtkui/callbacks.c
index 6160eae..24a3d2c 100644
--- a/plugins/gtkui/callbacks.c
+++ b/plugins/gtkui/callbacks.c
@@ -678,7 +678,7 @@ void
 on_changelog1_activate                 (GtkMenuItem     *menuitem,
                                         gpointer         user_data)
 {
-    show_info_window (DOCDIR "/ChangeLog", "DeaDBeeF " VERSION " ChangeLog", &changelogwindow);
+    show_info_window (DOCDIR "/changelog.deadbeef", "DeaDBeeF " VERSION " ChangeLog", &changelogwindow);
 }
 
 static GtkWidget *gplwindow;
