diff --git a/plugins/gtkui/callbacks.c b/plugins/gtkui/callbacks.c
index 0280c77..c670431 100644
--- a/plugins/gtkui/callbacks.c
+++ b/plugins/gtkui/callbacks.c
@@ -726,7 +726,7 @@ on_gpl1_activate                       (GtkMenuItem     *menuitem,
                                         gpointer         user_data)
 {
     char fname[PATH_MAX];
-    snprintf (fname, sizeof (fname), DOCDIR "/%s", "COPYING.GPLv2");
+    snprintf (fname, sizeof (fname), "/usr/share/common-licenses/%s", "GPL-2");
     show_info_window (fname, "GNU GENERAL PUBLIC LICENSE Version 2", &gplwindow);
 }
 
@@ -737,7 +737,7 @@ on_lgpl1_activate                      (GtkMenuItem     *menuitem,
                                         gpointer         user_data)
 {
     char fname[PATH_MAX];
-    snprintf (fname, sizeof (fname), DOCDIR "/%s", "COPYING.LGPLv2.1");
+    snprintf (fname, sizeof (fname), "/usr/share/common-licenses/%s", "LGPL-2.1");
     show_info_window (fname, "GNU LESSER GENERAL PUBLIC LICENSE Version 2.1", &lgplwindow);
 }
 
